src/cpu_timer.cpp

```diff
@@ -13,6 +13,10 @@
 // the library is being built (possibly exporting rather than importing code)
 #define BOOST_TIMER_SOURCE
 
+// define BOOST_CHRONO_HEADER_ONLY so that chrono dependencies are not
+// propagated to library users
+#define BOOST_CHRONO_HEADER_ONLY
```

Comment:
I believe Boost.Timer shouldn't define BOOST_CHRONO_HEADER_ONLY.

It is up to the user to define it.

This means that Boost.Timer should have something similar to Boost.Chrono so that the users of Boost.Timer can choose to have a Boost.Timer header only librray or not.